packages/taquito-contracts-library/src/rpc-wrapper.ts

```diff
@@ -321,4 +322,10 @@ export class RpcWrapperContractsLibrary implements RpcClientInterface {
   ): Promise<AllTicketBalances> {
     return this.rpc.getAllTicketBalances(contract, { block });
   }
+  async getOriginationProof(
+    reqs: OriginationProofParams,
```

Comment:
could we rename this? reqs doesn't seem like it has anything to do with what the value actually is